UnityProject/Assets/Scripts/Items/Implants/ImplantBase.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
```

Comment:
Codacy found an issue: [Remove this unnecessary 'using'.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6324510)